egov-persister/service-request-persister.yml

```
@@ -0,0 +1,138 @@
+            - jsonPath: $.Service.auditDetails.lastModifiedBy
+
+            - jsonPath: $.Service.auditDetails.createdTime
+
+            - jsonPath: $.Service.auditDetails.lastModifiedTime
+
+            - jsonPath: $.Service.additionalDetails
+              type: JSON
+              dbType: JSONB
+
+            - jsonPath: $.Service.accountId
+            
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)